src/components/POLine/const.js

```diff
@@ -45,3 +45,22 @@ export const INVENTORY_RECORDS_TYPE = {
   instanceAndHolding: 'Instance, Holding',
   none: 'None',
 };
+
+export const REQUIRED_LOCATION_QUANTITY_MAP_TO_INVENTORY = {
+  [INVENTORY_RECORDS_TYPE.all]: true,
+  [INVENTORY_RECORDS_TYPE.instance]: false,
+  [INVENTORY_RECORDS_TYPE.instanceAndHolding]: true,
+  [INVENTORY_RECORDS_TYPE.none]: false,
+};
+
+export const isLocationPhysicalQuantityRequired = (orderFormat, inventory) => (
```

Comment:
I'm not sure that file `const.js` should contain such service functions. IMHO, it's better to move their code to something like `validation.js` along with LocationForm files